site/components/page-templates/UserDetailPageTemplate.tsx

```diff
@@ -32,6 +36,69 @@ const UserDetailPageTemplate = ({
     setPageState,
 }: Props) => {
     const stateUpdater = getStateUpdater(setPageState, pageState);
+    const [selectedOperator, setSelectedOperator] = useState<SingleValue<OperatorOrgSchema>>(
+        pageState.inputs.operatorOrg ?? null,
+    );
+    const [operatorSearchInput, setOperatorsSearchInput] = useState<string>("");
+
+    const operatorsListForOrg = pageState.operatorsForOrg ?? [];
+
+    const handleOperatorChange = (value: SingleValue<OperatorOrgSchema>) => {
+        const parsed = operatorOrgSchema.safeParse(value);
+
+        if (!parsed.success) {
+            setPageState({
+                ...pageState,
+                errors: [
+                    ...pageState.errors.filter((err) => !Object.keys(addUserSchema.shape).includes(err.id)),
+                    ...flattenZodErrors(parsed.error),
+                ],
+            });
+        } else {
+            setSelectedOperator(parsed.data);
+            setPageState({
+                ...pageState,
+                inputs: {
+                    ...pageState.inputs,
+                    operatorOrg: parsed.data,
+                },
+                errors: [...pageState.errors.filter((err) => !Object.keys(addUserSchema.shape).includes(err.id))],
+            });
+        }
+    };
+
+    const removeOperator = (e: SyntheticEvent) => {
+        e.preventDefault();
+
+        setPageState({
+            ...pageState,
+            inputs: {
+                ...pageState.inputs,
+                operatorOrg: undefined,
+            },
+            errors: pageState.errors,
+        });
+        setSelectedOperator(null);
+    };
+
+    const getOperatorRows = () => {
+        if (pageState.inputs.operatorOrg) {
+            return [pageState.inputs.operatorOrg].map((operator: OperatorOrgSchema) => ({
```

Comment:
getOperatorRows and removeOperator are functions used for creating the table of selected operators. We don't actually need this at all so I've removed these functions.

To remove getOperatorRows I had to refactor the searchSelect component so that getRows doesn't have to be provided to it.